<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>输入演员网址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="url_input_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="go_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="log_area">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>331</width>
     <height>152</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="update_check">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>是否更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
